Dockerfile.win64

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.7.6
+
+MAINTAINER Ivan Kozlovic <ivan.kozlovic@apcera.com>
+
+COPY . /go/src/github.com/nats-io/gnatsd
+WORKDIR /go/src/github.com/nats-io/gnatsd
+
+RUN CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -ldflags "-s -w" -o gnatsd.exe .
+
```

Comment:
This Dockerfile is just used to build the executable. The executable will be extracted from `nats-docker/update.sh`  and copied in respective `nats-docker/windows` sub-directories. This repo has not yet been updated.